README.md

```diff
@@ -90,13 +97,31 @@ You can use a [PAT](https://help.github.com/en/github/authenticating-to-github/c
 
 ```yml
       - name: Slash Command Dispatch
-        uses: peter-evans/slash-command-dispatch@v1
+        uses: peter-evans/slash-command-dispatch@v2
         with:
-          token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          reaction-token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          commands: deploy, integration-test, build-docs
+          token: ${{ secrets.PAT }}
+          reaction-token: ${{ secrets.PAT }}
+          commands: |
+            deploy
+            integration-test
+            build-docs
 ```
 
+#### `dispatch-type`
+
+By default, the action creates [repository_dispatch](https://developer.github.com/v3/repos/#create-a-repository-dispatch-event) events.
+Setting `dispatch-type` to `workflow` will instead create [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) events.
+There are significant differences in the action's behaviour when using `workflow` dispatch. See [workflow dispatch](docs/workflow-dispatch.md) for usage details.
+
+For the majority of use cases, the default `repository` dispatch will likely be the most suitable for new workflows.
+If you already have `workflow_dispatch` workflows, you can execute them with slash commands using this action.
+
+| Repository Dispatch (default) | Workflow Dispatch |
```

Comment:
in the github docs, I can see the note

> Note: This event will only trigger a workflow run if the workflow file is on the master or default branch.

does this mean, when I initially setup slash commands for my repo..
- when using  `repository_dispatch` the slash commands only work after the PR has merged (so I can only test it after merging in the default-branch)
- but when using `workflow_dispatch` I can already test it in the PR without the need to merge it beforehand?

Reply:
Yes, exactly. `workflow_dispatch` workflows will be executable in a PR branch before merge. You can use the `ref` named argument to target the PR branch name.

Reply:
I feel this is a major difference and should be noted

Reply:
It is a good benefit. Updated.